airflow/example_dags/example_jdbc_operator.py

```diff
@@ -0,0 +1,66 @@
+#
```

Comment:
This file should be in `airflow.providers.jdbc.example_dags`

Reply:
moved